extractjava.py

```diff
@@ -30,6 +30,8 @@
                 try:
                     e.decode("utf8")
                     if len(e) > 1000:
+                        # Exclude massively long fragments to cut down the
```

Comment:
I think "massively" is overstating it! Can we make this a constant somewhere at the top and make clear what the units are (characters?)?

Reply:
Fixed in latest commit.